docs/self-hosted/quickstart.md

```diff
@@ -1,158 +1,64 @@
 ---
 description:
-  If you're looking for the fastest way to get up-and-running with Directus, this guide will walk you through getting
-  things installed, configured, and modeled.
-readTime: 5 min read
+  If you're looking for the fastest way to get up-and-running with Directus locally, this guide will get you there in minutes.
 ---
 
-# Quickstart Guide
+# Self-Hosting Quickstart
 
-> If you're looking for the fastest way to get up-and-running with Directus, this guide will walk you through getting
-> things installed, configured, and modeled.
+<iframe style="width:100%; aspect-ratio:16/9; margin-top: 2em;" src="https://www.youtube.com/embed/-UASj_6WmMQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-## Requirements
+## Install Docker
 
-- [Docker](https://docs.docker.com/get-docker/)
-- [Docker Compose](https://docs.docker.com/compose/install/) (often included with newer Docker installations)
+You should have [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
 
-It can be easy to under-provision resources to run a self-hosted instance of Directus. For Directus' container resources, the required minimum system requirements are 1x 0.25 vCPU / 512 MB, although the recommended minimum is 2x 1 vCPU / 2GB.
+:::info What Is Docker?
 
-## 1. Installation
+Docker is a developer tool that allows software-creators to distribute their work along with all dependencies and required environment settings. This means that applications can run reliably and consistently, making it the perfect way to use Directus both locally and in-production.
 
-You can use the following configuration to get started using Docker Compose. Make sure to change all sensitive values
-like `KEY`, `SECRET`, `ADMIN_PASSWORD`, _etc._
+As soon as there are new releases of Directus, we publish them on [Docker Hub](https://hub.docker.com/r/directus/directus).
 
-```yaml
+:::
+
+## Create a Docker Compose File
+
+Create a new empty directory, and open it in a text editor. Create a `docker-compose.yml` file and paste the following:
+
+```yml
 version: '3'
 services:
   directus:
     image: directus/directus:latest
     ports:
       - 8055:8055
     volumes:
-      - ./uploads:/directus/uploads
       - ./database:/directus/database
+      - ./uploads:/directus/uploads
```

Comment:
Might also be worthh mentioning here that the relative patch is relative to the current location you're starting the docker from.

Reply:
Mentioned in the quickstart so gonna resovle this one